src/js/tech/html5.js

```diff
@@ -282,6 +342,63 @@ class Html5 extends Tech {
     this.textTracks().removeTrack_(e.track);
   }
 
+  proxyNativeVideoTracks_() {
+    let vt = this.el().videoTracks;
+
+    if (vt && vt.addEventListener) {
+      vt.addEventListener('change', this.handleVideoTrackChange_);
+      vt.addEventListener('addtrack', this.handleVideoTrackAdd_);
+      vt.addEventListener('removetrack', this.handleVideoTrackRemove_);
+    }
+  }
+
+  handleVideoTrackChange(e) {
+    let vt = this.videoTracks();
+    this.videoTracks().trigger({
+      type: 'change',
+      target: vt,
+      currentTarget: vt,
+      srcElement: vt
+    });
+  }
+
+  handleVideoTrackAdd(e) {
+    let track = e.track;
+    if(!(e.track instanceof VideoTrack)) {
```

Comment:
native Video and Audio Tracks won't be inherit from eventhandler so we have to use them to create a VideoTrack of our own that does inherit from eventhandler so that we can watch for events regardless of wether it is native or not.